cartographer/mapping_2d/local_trajectory_builder.cc

```diff
@@ -73,8 +79,19 @@ const Submaps* LocalTrajectoryBuilder::submaps() const { return &submaps_; }
 sensor::LaserFan LocalTrajectoryBuilder::BuildCroppedLaserFan(
     const transform::Rigid3f& tracking_to_tracking_2d,
     const sensor::LaserFan& laser_fan) const {
+  LaserFan fan{laser_fan.origin, {}, {}, laser_fan.reflectivities};
```

Comment:
Should reflectivities be retained here? And if so: Do we need to drop/reorder reflectivities when elements are dropped from 'returns'?